src/book/03-guides/01-main-codebase/04-robocup-setup.mdx

```diff
@@ -0,0 +1,27 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Robocup Setup
+description: Setting up the robot for robocup
+slug: /guides/main/getting-started
+authors:
+  - Ysobel Sims (@ysims)
+  - Josephus Paye II (@JosephusPaye)
+  - Mikyla Peters (@miikyla)
+  - Lachlan Court (@LachlanCourt)
+---
+
+## Networking
+
+1. `cd system/nugusX/etc/systemd/network/` to go to the config folder that system configuration will use for networkd.
+2. Rename `40-wlp58s0-robocup.network` to `30-wlp58s0-robocup.network`
+3. Rename `30-wlp58s0.network` to `40-wlp58s0.network`
```

Comment:
You could just rename the RoboCup one to be lower than 30 rather than renaming both. Also I think both the old 30 file and the new 30 file will both be around still, and it'll just be lexicographical order of which one runs.

Also it might be worth adding something as to how to undo this after RoboCup, since both of these files will be left behind and you'll need to put it back in "Lab" mode

Reply:
I think the best way might be to swap the two files rather than changing their numbers actually...

Reply:
Swapping them will be good, the system configuration tool does not delete any old files in these directories so they will stay around on the robot.